lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -161,7 +161,7 @@ describe('CategoryRenderer', () => {
         '.lh-clump--not-applicable .lh-audit-group__summary'));
 
     const notApplicableCount = a11yCategory.auditRefs.reduce((sum, audit) =>
-        sum += audit.result.scoreDisplayMode === 'not-applicable' ? 1 : 0, 0);
+        sum += audit.result.scoreDisplayMode === 'notApplicable' ? 1 : 0, 0);
```

Comment:
we need the renderer to handle both. so these will have to check both casings. 
Maybe make a util method?

Reply:
aren't we fixing all of them before they make it this far though?

Reply:
> aren't we fixing all of them before they make it this far though?

yeah, @paulirish it gets updated in `Util.prepareReportResult` so it's always `'notApplicable'` before it gets passed in to the renderer.

Reply:
ah sry. yes good call.